release notes/v0.55.0.md

```diff
@@ -0,0 +1,215 @@
+k6 `v0.55.0` is here 🎉! This release includes:
+
+- ⚠️ The deprecated StatsD output has been removed.
+- ⚠️ The experimental `k6/experimental/tracing` module has been removed.
+- 🆕 URL grouping support in the browser module.
+- 🆕 Top-level `await` support.
+- 🔐 Complete RSA support for `k6/experimental/webcrypto`.
+
+## Breaking changes
+
+### `k6/experimental/tracing` module removed [`#3855`](https://github.com/grafana/k6/pull/3855)
+
+The experimental `k6/experimental/tracing` module has been removed, in favor of a replacement [jslib](https://jslib.k6.io/http-instrumentation-tempo/) polyfill, please consult our [guide on how to migrate](https://grafana.com/docs/k6/latest/javascript-api/jslib/http-instrumentation-tempo/#migration-from-k6experimentaltracing), [`#3855`](https://github.com/grafana/k6/pull/3855).
+
+### StatsD output removed [`#3849`](https://github.com/grafana/k6/pull/3849)
+
+The StatsD output was deprecated in k6 v0.47.0 and is now removed. You could still output results to StatsD using the community xk6 extension [LeonAdato/xk6-output-statsd](https://github.com/LeonAdato/xk6-output-statsd). Thanks, @LeonAdato for taking over the extension!
+
+### `open` will have a breaking change in the future.
+
+Currently, `open` opens relative files based on an unusual root, similar to how `require` behaved before it was updated for ESM compatibility. To make k6 more consistent, `open` and other functions like it will start handling relative paths in the same way as imports and `require`.
+For a more in-depth explanation, please refer to the related [issue](https://github.com/grafana/k6/issues/3857).
+
+With this version, k6 will start emitting warnings when it detects that in the future, this will break. We recommend using [`import.meta.resolve()`](https://grafana.com/docs/k6/latest/javascript-api/import.meta/resolve/) as a way to make your scripts future proof.
+
```

Comment:
### `http.file#data` now truly has the same type as the provided data [`#4009`](https://github.com/grafana/k6/pull/4009)

Previously `http.file#data` was always a slice of byte (`[]byte`) - which was very likely a bug and a leftover from years past.

Both it's documentation, and it is idea though, was to be the original data that was provided when creating it.